src/Illuminate/Contracts/Validation/Validator.php

```diff
@@ -48,6 +48,17 @@ public function failed();
      */
     public function sometimes($attribute, $rules, callable $callback);
 
+    /**
+     * Validate the new rules if the previous attribute have been validated.
+     *
+     * @param  string|array  $attribute
+     * @param  array  $rules
+     * @param  array  $messages
+     * @param  array  $customAttributes
+     * @return $this
+     */
+    public function validateIfPasses($attributes, array $rules, array $messages = [], array $customAttributes = []);
```

Comment:
We cannot add methods to contracts on patch releases.

Reply:
I changed the base branch to 10.x

Reply:
I changed the base branch to 10.x